packages/bpk-component-swap-button/src/BpkSwapButton.module.scss

```
@@ -0,0 +1,63 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '../../bpk-mixins/index.scss';
+
+@layer bpk-component {
+  .bpk-swap-button {
+    width: bpk-spacing-xxl();
+    height: bpk-spacing-xxl();
+    transform: translateY(-50%) rotate(90deg);
+
+    @include bpk-breakpoint-small-tablet {
+      transform: rotate(0deg);
+    }
+
+    :focus-visible {
+      box-shadow:
+        0 0 0 (bpk-spacing-sm() / 2) $bpk-color-white,
+        0 0 0 bpk-spacing-sm() $bpk-text-link-day;
+    }
+
+    &__button {
+      display: flex;
+      width: 100%;
+      height: 100%;
+      justify-content: center;
+      align-items: center;
+      transition: transform $bpk-duration-base;
+      border: $bpk-border-size-xl solid $bpk-core-primary-day;
+      border-radius: 50%;
+      background-color: $bpk-color-white;
```

Same here, this would be

```suggestion
      background-color: $bpk-surface-default-day;
```